client/app/components/dialog-dashboard/dialog-dashboard.html

```diff
@@ -0,0 +1,52 @@
+<div class='dialog'>
+  <div style="background-color: #f1f1f1; padding: 20px;"
+       ng-model='dialogDashboard.dialogTabs[dialogDashboard.service.activeTab].dialog_groups'
+       ui-sortable='dialogDashboard.sortableOptionsBox'
+       ng-repeat='tab in dialogDashboard.dialogTabs'
+       ng-show='tab.position == dialogDashboard.service.activeTab'>
+    <!-- tab content -->
+    <div ng-repeat='box in tab.dialog_groups track by $index'>
+      <div class="panel panel-default"
+           data-drop='true'
+           jqyoui-droppable='{multiple: true, onDrop: "dialogDashboard.droppableOptions"}'
+           ng-model='box.dialog_fields'>
+        <div class="panel-heading">
+          <strong>{{ box.label }}</strong>
+          <button type="button" class="close"
+                  ng-click="dialogDashboard.removeBox(box.position)">
+            <span aria-hidden="true">
+              <i class="fa fa-times"></i>
+            </span>
+          </button>
+        </div>
+        <div class="panel-body">
+          <div ui-sortable='dialogDashboard.sortableOptionsFields'
+               ng-model='box.dialog_fields'>
+            <div class="well well-lg nosort"
+                 ng-if='box.dialog_fields.length === 0' translate>
+              <i class="fa fa-object-group"></i>Drag your components here
+            </div>
+            <div ng-repeat='field in box.dialog_fields'>
+              <form class="form-horizontal">
+                <dialog-field box-position="box.position" field-data='field'></dialog-field>
+              </form>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="blank-slate-pf nosort"
+         ng-if='tab.dialog_groups.length === 0'
+         ng-click='dialogDashboard.addBox()'>
+      <div class="blank-slate-pf-icon">
+        <i class="fa fa-plus-circle"></i>
+      </div>
+      <h1 translate>Start with adding a box</h1>
+    </div>
+    <div class="panel panel-default nosort">
+      <a ng-click='dialogDashboard.addBox()' translate>
```

Comment:
`translate` .. will it pick just that "Create Box" or will it pick up the whole "<i class...Box"?